src/main/webapp/index.html

```diff
@@ -34,36 +34,38 @@
           <span class="navbar-toggler-icon"></span>
         </button>
         <div class="collapse navbar-collapse" id="navbarSupportedContent">
-          <input
-            class="form-control mr-2"
-            type="search"
-            name="searchValue"
-            id="searchValue"
-            placeholder="Search by Title"
-            oninput="getDashboardItems()"
-          />
-          <select
-            class="form-control mr-2 w-auto"
-            name="sortingDirection"
-            id="sortingDirection"
-            onchange="getDashboardItems()"
-            required
-          >
-            <option value="ASCENDING" selected
-              >Increasing Alphabetical Order (Title)</option
+          <div class="form-inline my-2 my-lg-0">
```

Comment:
Nit: The spacing is  far (I think 4 spaces). This should be an easy fix.

Reply:
I formatted the file with prettier before committing it, I think that there are 2 spaces instead of 4 right now.